soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -165,7 +165,16 @@ endchoice
 
 choice SOC_HFXO_LOAD_CAPACITANCE
 	prompt "HFXO load capacitance"
-	default SOC_HFXO_CAP_EXTERNAL
+	default SOC_HFXO_CAP_DEFAULT
+
+config SOC_HFXO_CAP_DEFAULT
+	bool "Use default SoC settings"
```

Comment:
"Use default SoC settings" will be displayed in the menuconfig. It may be difficult for the user to infer what the setting is

Reply:
That's why I provide the help message for this option. The actual default setting for a given SoC depends on what is contained in the `NRF_FICR_S->TRIMCNF[n]` registers.

Reply:
I'll change the prompt to "SoC default" as suggested by @tejlmand in https://github.com/nrfconnect/sdk-zephyr/pull/561#discussion_r652652609.